hr_expense_petty_cash/__manifest__.py

```diff
@@ -3,9 +3,9 @@
 
 {
     "name": "Petty Cash",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "category": "Human Resources",
-    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "author": "Trinityroots, Odoo Community Association (OCA)",
```

Comment:
**Don't** delete author.
you can add it, if you implement this module to big change.